2018/_data/speakers.json

```diff
@@ -206,6 +206,17 @@
       "ref": "SW9udXRCYWxvc2luaW9udXRiYWxvc2luQHlhaG9vLmNvbQ==",
       "url": "infoSpeaker.html?ref=SW9udXRCYWxvc2luaW9udXRiYWxvc2luQHlhaG9vLmNvbQ==",
       "twitter": "ionutbalosin"
-    }
+    },
+    {
+      "enabled": "1",
+      "name": "Mauricio Salatino",
+      "description": "",
```

Comment:
I'd suggest to include his "title" or role like we have with the rest of speakers

Reply:
Done